jobs/haproxy/spec

```diff
@@ -553,6 +554,9 @@ properties:
   ha_proxy.accept_proxy:
     description: "Turned off by default. Enforces the use of the PROXY protocol for all incoming connections to all frontends, with the exception of local requests to the health check endpoint, since Monit does not support PROXY protocol. When enabled, standard TCP connections to these ports no longer work."
     default: false
+  ha_proxy.expect_proxy:
+    description: "Enables proxy protocol for the specified list of CIDR ranges. Only applies if `ha_proxy.accept_proxy` is disabled."
+    default: ""
```

Comment:
if you put `default: ~`, it will not be set by default. The `spec` file is used to set default values in the manifest, if they're absent.

For the logic, I really you'd check for a list, and if `expect_proxy` is `nil` (because the value is not set), then you know that it's not active. This is particularly important for the mutual exclusivity with the `accept_proxy` property.